pavelib/prereqs.py

```diff
@@ -196,21 +196,11 @@ def uninstall_python_packages():
     for _ in range(3):
         uninstalled = False
         frozen = sh("pip freeze", capture=True).splitlines()
```

Comment:
Given you're only using `frozen` to check for contents, it seems easier to not do splitlines, but get it back as a full string. That way you can do `"{}==".format(package) in frozen` which seems easier in a big string, rather than iterating over a list of strings.